pkg/api/maestro_instance.go

```diff
@@ -0,0 +1,14 @@
+package api
+
+// Instance is used by Maestro for discovering running peers,
+// but it is not intended for direct exposure to end users via the API.
+type Instance struct {
```

Comment:
ServerInstance? Should add more descriptions, e.g. what a instance represents and I think we use it to maintain the liveness of the maestro server, which field will be used to track?

Reply:
Added the description as suggested, especially make it clear that `updateAt` field is used to track the liveness of server instance.